roles/eda/templates/eda.configmap.yaml.j2

```
@@ -11,6 +11,9 @@ data:
   # Operator specific settings
   EDA_DEPLOYMENT_TYPE: "k8s"
 
+  # Public URL
+  EDA_WEBHOOK_BASE_URL: "{{ public_base_url }}"
```

This is wrong. As [I commented in my first review](https://github.com/ansible/eda-server-operator/pull/230#discussion_r1705991192), it must contain the prefix_path that is configured for eda. 
Please @msmagnanijr can you just use this code? 

```suggestion
  EDA_WEBHOOK_BASE_URL: "{{ public_base_url.rstrip('/') }}/{{ eda_webhook_prefix_path.lstrip('/') }}"
```